.travis.yml

```diff
@@ -1,24 +1,22 @@
-sudo: required
-
 language: bash
+services: docker
 
-services:
-  - docker
-
-install:
-  - docker build -t quay.io/azavea/django:${TRAVIS_COMMIT:0:7} .
+env:
+  - VERSION=1.9 PYTHON_VERSION=2.7 PG_MAJOR=9.5 PG_VERSION=9.4+165+deb8u2 VARIANT=slim
```

Comment:
I know its less of an issue since the package versions are baked into the image and not re-downloaded on every build, unlike Vagrant+Ansible builds, but should we use wildcards in the PG_VERSION variable to avoid extra maintenance of occasionally having to fix these version strings?